examples/c/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(CMAKE_CXX_STANDARD 20)
 option(USE_CUDA "Build with CUDA support" OFF)
 option(USE_CXX "Invoke the C++ example" ON)
 option(PHI3 "Build the Phi example" OFF)
+option(LLAMA "Build the Llama example" OFF)
```

Comment:
Could we have something like the following?

```suggestion
option(LLM "Build the large-language model example" OFF)
option(VLM "Build the vision-language model example" OFF)
option(ALM "Build the audio-language model example" OFF)
```

The abbreviations and names can be different, but the idea would be to create examples grouped by input and output modality.
- The `LLM` could be for decoder-only models (e.g. LLaMA, Phi)
  - Inputs: text
  - Outputs: text
- The `VLM` could be for vision-text models (e.g. Phi-3/Phi-3.5 vision)
  - Inputs: images, text
  - Outputs: text
- The `ALM` could be for audio-text models (e.g. Whisper)
  - Inputs: audios, text
  - Outputs: text

Reply:
I think this is a great proposal for a new PR!